README.md

```diff
@@ -47,6 +47,7 @@ from fastapi import FastAPI
 from freddie.exceptions import NotFound
 from freddie.schemas import Schema
 from freddie.viewsets import FieldedViewset, PaginatedListViewset, route, ViewSet
+from typing import List
```

Comment:
move this import before `from fastapi import FastAPI`

Reply:
fixed